onnx/runtime/aionnx/_op.py

```diff
@@ -0,0 +1,198 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any, Dict
+
+import numpy as np  # type: ignore
+
+from ...onnx_pb import NodeProto
+from ..op_run import OpRun, RuntimeTypeError
+
+
+class OpRunUnary(OpRun):  # pylint: disable=W0223
+    """
+    Ancestor to all unary operators in this subfolder.
+    Checks that inputs type are the same.
```

Comment:
Don't understand this. Unary operators have only one input, right?

Reply:
The comment was wrong. I updated it. It checks that input and output types are the same.